apps/src/templates/instructions.html.ejs

```diff
@@ -14,3 +16,19 @@
 <% if (locals.aniGifURL) { -%>
   <img class="aniGif example-image" src='<%= locals.aniGifURL %>'/>
 <% } -%>
+
+<% if (locals.authoredHints && locals.authoredHints.length) { -%>
+<h4 class="hint">
+  <details>
+    <summary role="button" aria-expanded="true"><b>Hints</b></summary>
+    <% for (var i = 0; i < locals.authoredHints.length; i++) { %>
+      <h5 class="hintInner">
+        <details>
+          <summary role="button" aria-expanded="false"><b>Hint #<%- i %></b></summary>
+          <%- locals.authoredHints[i] %>
+        </details>
+      </h5>
+    <% } %>
+  </details>
+</h4>
+<% } %>
```

Comment:
Another case where I don't know for sure whether this would start to have weird interactions with our jQuery code if we converted to React, but particularly as this ejs gets more complicated my preference would be for React if possible.